index.html

```diff
@@ -0,0 +1,379 @@
+<!DOCTYPE html>
+<html lang="en-ca">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="A network of independent designers, researchers, developers, and engineers.">
+        <meta name="keywords" content="design, ui, ux, mobile, front-end, html, css, interface, vancouver, canada">
+        <meta name="author" content="Ben Groulx, Sean Durfee">
+        <title>House House Design • Digital Design & Strategy</title>
+        <style>
+            :root {
+                --color-background-01: #534D99;
+                --color-background-02: #4F3275;
+                --color-background-03: #195580;
+                --color-background-04: #19806D;
+                --color-action: #FF9E68;
+                --color-text: #FEFEFE;
+                --color-text-faded: rgba(255, 255, 255, 0.6);
+                --space-xs: 0.2em;
```

Comment:
1. Can we round this to .25 so we get whole px values?
2. Was the use of `em` instead of `rem` intentional?